### DISCUSSION TITLE
What was the first video uploaded to YouTube?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#801] [Galaxy Trivia] [answered]
**Bonus:** Pin the exact location of the person in the video to the map.

**Comment by fathiputera** [comment]
The first video uploaded to YouTube is titled "Me at the zoo" and was uploaded by Jawed Karim, one of the co-founders of YouTube. The video was uploaded on April 23, 2005. In the video, Jawed Karim is seen standing in front of the elephant enclosure at the San Diego Zoo.

https://goo.gl/maps/FWTL9wBjR7d1PXK89?coh=178572&entry=tt

**Comment by peterbaccaro** [comment, marked as answer]
"**Me at the zoo**" is the first ever video uploaded to YouTube, on April 23, 2005, 8:31:52 p.m. PDT, or April 24, 2005, at 03:31:52 UTC. The 19-second video features YouTube's co-founder Jawed Karim, who was 25 years old at the time, in front of two elephants at the San Diego Zoo in California, noting their long trunks.

The exact location would be Elephant Odyssey at San Diego Zoo in California:
- https://goo.gl/maps/uWgJqmAB5hwbqTJj9?coh=178573&entry=tt

**Comment by SANKARAMDAS** [comment]
Me at the zoo
"Me at the zoo" is the first ever video uploaded to YouTube, on April 23, 2005, 8:31:52 p.m. PDT, or April 24, 2005, at 03:31:52 UTC. The 19-second video features YouTube's co-founder Jawed Karim, who was 25 years old at the time, in front of two elephants at the San Diego Zoo in California, noting their long trunks.

https://en.wikipedia.org/wiki/Me_at_the_zoo#:~:text=%22Me%20at%20the%20zoo%22%20is,California%2C%20noting%20their%20long%20trunks.
